macros/WebExtAllExamples.ejs

```diff
@@ -0,0 +1,60 @@
+
+<%
+
+/*
+The "examples.json" file from the https://github.com/mdn/webextensions-examples
+repository contains information about the examples living in that repository.
+
+This macro loads the "examples.json" file and uses it to build a table
+presenting that information.
+*/
```

Comment:
I like this intro comment, and would like to see this standard on macros.

Since this macro is basically for a single page, it would be good to include the URL https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Examples in the comment.